src/test/java/uk/gov/digital/ho/pttg/SmokeTest.java

```diff
@@ -17,30 +27,28 @@
     private final String IP_API_PATH = "/incomeproving/v3/individual/financialstatus";
 
     private String ipApiRootUrl;
+    private String ipApiAuth;
+
     private SimpleHttpClient simpleHttpClient;
-    private Gson gson;
+    private ObjectMapper objectMapper;
 
     @Before
     public void setUp() {
         ipApiRootUrl = getMandatoryEnvVar("IP_API_ROOT_URL");
-        simpleHttpClient = new SimpleHttpClient();
-        gson = new GsonBuilder().registerTypeAdapter(LocalDate.class, new LocalDateSerializer())
-                                .create();
+        ipApiAuth = getMandatoryEnvVar("IP_API_AUTH");
+        simpleHttpClient = new SimpleHttpClient(Collections.singletonMap(ipApiRootUrl, ipApiAuth));
+        objectMapper = initialiseObjectMapper(new ObjectMapper());
```

Comment:
Could the ObjectMapper be reused across all the tests, rather than instantiate for each one?

Reply:
Yes - I'll change it